tests/scf-bs/input.dat

```diff
@@ -11,6 +11,7 @@ H 1 3.0
 symmetry c1
 }
 set reference uhf
+set guess gwh
```

Comment:
So this is a UHF calculation on H2 close to dissociation.

I looked at the code, and the symmetry breaking should also work with SAD, since it is done when the orbitals are formed for the first time.

What must be happening here is that the HOMO and LUMO are of the same symmetry in the SAD guess, so mixing these orbitals does not give you the wanted broken symmetry.